test/index.js

```diff
@@ -3,6 +3,7 @@
 import test from 'tape';
 import irecord from '../index.js';
 import rx from './rx';
+import stampit from './stampit';
```

Comment:
I think it's warning that they're unused because they're not actually used in this file. I did a search myself. ESLint is right. ;)